administrator/language/de-DE/de-DE.com_modules.ini

```diff
@@ -7,33 +7,39 @@
 COM_MODULES="Module"
 COM_MODULES_ACTION_EDITFRONTEND="Frontend-Bearbeitung"
 COM_MODULES_ACTION_EDITFRONTEND_COMPONENT_DESC="Erlaubt es Benutzern dieser Gruppe, Module direkt im Frontend zu bearbeiten."
-COM_MODULES_ADMIN_LANG_FILTER_FIELDSET_LABEL="Administratormodule"
+COM_MODULES_ADD_CUSTOM_POSITION="Eigene Position hinzufügen"
 COM_MODULES_ADMIN_LANG_FILTER_DESC="Ermöglicht das Filtern von Administratormodulen pro Administratorsprache."
 COM_MODULES_ADMIN_LANG_FILTER_LABEL="Sprachfilterung"
 COM_MODULES_ADVANCED_FIELDSET_LABEL="Erweitert"
 COM_MODULES_ASSIGNED_VARIES_EXCEPT="Alle, außer ausgewählte"
 COM_MODULES_ASSIGNED_VARIES_ONLY="Ausgewählte"
 COM_MODULES_BASIC_FIELDSET_LABEL="Optionen"
+COM_MODULES_BATCH_OPTIONS="Mehrere ausgewählte Module gleichzeitig bearbeiten (Stapelverarbeitung)"
 COM_MODULES_BATCH_POSITION_LABEL="Position setzen"
 COM_MODULES_BATCH_POSITION_NOCHANGE="Ursprüngliche Position beibehalten"
 COM_MODULES_BATCH_POSITION_NOPOSITION="Keine Modulposition"
-COM_MODULES_BATCH_OPTIONS="Mehrere ausgewählte Module gleichzeitig bearbeiten (Stapelverarbeitung)"
 COM_MODULES_BATCH_TIP="Wenn ein Modul kopiert wird, so werden auch alle zusätzlich ausgewählten Aktionen auf das kopierte Modul angewendet. Ansonsten werden die Aktionen nur auf das ausgewählte Modul angewendet.<br />Beim Kopieren ohne die Position des Moduls zu ändern, ist es trotzdem notwendig „Ursprüngliche Position beibehalten“ auszuwählen."
 COM_MODULES_CHANGE_POSITION_BUTTON="Auswählen"
 COM_MODULES_CHANGE_POSITION_TITLE="Wechseln"
+COM_MODULES_COLLAPSE="Einklappen"
 COM_MODULES_CONFIGURATION="Module: Optionen"
 COM_MODULES_CUSTOM_OUTPUT="Benutzerdefinierte Ausgabe"
-COM_MODULES_ERR_XML="Modul-XML-Daten nicht verfügbar"
+COM_MODULES_CUSTOM_POSITION="Aktive Positionen"
+COM_MODULES_DESELECT="Abwählen"
 COM_MODULES_ERROR_CANNOT_FIND_MODULE="Modul kann nicht gefunden werden"
 COM_MODULES_ERROR_CANNOT_GET_MODULE="Modul kann nicht geladen werden"
 COM_MODULES_ERROR_INVALID_EXTENSION="Ungültiges Modul"
 COM_MODULES_ERROR_NO_MODULES_SELECTED="Kein Modul ausgewählt"
+COM_MODULES_ERR_XML="Modul-XML-Daten nicht verfügbar"
+COM_MODULES_EXPAND="Ausklappen"
 COM_MODULES_EXTENSION_PUBLISHED_DISABLED="Modul deaktiviert und veröffentlicht"
 COM_MODULES_EXTENSION_PUBLISHED_ENABLED="Modul aktiviert und veröffentlicht"
 COM_MODULES_EXTENSION_UNPUBLISHED_DISABLED="Modul deaktiviert und versteckt"
 COM_MODULES_EXTENSION_UNPUBLISHED_ENABLED="Modul aktiviert und versteckt"
 COM_MODULES_EXTRA_STYLE_DESC="Den Template-Stil für das Modul oder die Module in der ausgewählten Position überschreiben."
 COM_MODULES_EXTRA_STYLE_TITLE="Modulstil"
+COM_MODULES_FIELDSET_RULES="Modulberechtigungen"
+COM_MODULES_FIELD_BOOTSTRAP_SIZE_LABEL="Bootstrap-Größe"
```

Comment:
do we use that at other places too?

Reply:
just reorder... no new translation added